app/mailers/school_mailer.rb

```diff
@@ -326,4 +327,14 @@ def nqt_plus_one_sit_invite(recipient:, start_url:, school:)
       },
     ).tag(:year2020_invite).associate_with(school, as: :school)
   end
+
+  def diy_wordpress_notification(user:)
+    template_mail(
+      DIY_WORDPRESS_NOTIFICATION_TEMPLATE,
+      to: user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {},
+    ).tag(:diy_wordpress_notification).associate_with(user, as: :user)
```

Comment:
No need to associate with the user, it will be automatically associated as recipient (unless he changes his email between email being scheduled and sent).

However we might want to associate this email with a school as well?

Reply:
There isn't necessarily a single school here, the SIT may be controlling two relevant schools - we would just send one email

Reply:
Ah, makes sense then. However I keep thinking about that linking - there's usually quite a lot of objects we want to link email to! For example, this email might be linked to all the schools of given SIT and to his SIT profile as well. I try to imagine that there is a small link in the admin panel called "related emails" and wonder what emails should be displayed there for each object....

Reply:
I've removed the explicit link